src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,87 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+
+@Entity
+public class User {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  @Column(nullable = false)
+  private String username;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private LocalDate birthdate;
+
+  @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
This cascade is wrong. The books exist independantly of the user, and a book could be asociated to more than one user. Do you remember what CascadeType combination is suitable for collections where the only thing you want is to mantain the jointable updated?

Reply:
You're right!